SOP/SOP-push.adoc

```diff
@@ -0,0 +1,179 @@
+. Check the environment variables of the Database
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-postgresql-<xyz123> -> Environment Variables`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the logs of the UPS OAuth Proxy Container
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-<xyz123> -> Logs -> Container -> ups-oauth-proxy`
+.. Check the `pod/example-unifiedpushserver-<xyz123>` events, see if was possible to pull the OAuth image.
+.. Check the logs of the UPS Container
+.. Go to : `Applications -> Pods -> example-unifiedpushserver-<xyz123> -> Logs -> Container -> ups`
+. Check the `pod/example-unifiedpushserver-<xyz123>` events, see if was possible to pull the UnifiedPush image.
+. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. Check if the secret was created (It cannot be checked by the UI)
+. Check if the values in the secret are correct (It cannot be checked by the UI)
+
+NOTE: If resolving the UnifiedPushOperatorDown doesn't resolve the issue, please continue with the below steps
+
+==== CLI
+
+. Check that the UnifiedPushServer CR is deployed in the same namespace as the operator
+. Check the environment variables of the Server
+.. Run `oc describe pod/example-unifiedpushserver-<xyz123>`
+.. For further information see https://github.com/aerogear/aerogear-unifiedpush-server#container-configuration.
+.. Note that it will use the values mapped in the Secret created by the operator with the database pod name.
+. Check the environment variables of the Database
+.. Run `oc describe pod/example-unifiedpushserver-postgresql-<xyz123>`
+.. By default it should use the values defined in the Secret created by the operator with the database pod name.
+. Check the logs of the UPS OAuth Proxy Container
+.. Get the service pod name -> `oc get pods | grep unifiedpushserver`
+.. Run `oc logs <service-podname> -c ups-oauth-proxy`
+.. Save the logs by running `oc logs <service-podname> -c ups-oauth-proxy > <filename>.log`
+.. Check if was possible pull the OAuth image
+. Check the logs of the UPS Container
+.. Run `oc logs <service-podname> -c ups`
+.. Save the logs by running `oc logs <service-podname> -c ups > <filename>.log`
+.. See the `pod/example-unifiedpushserver-<xyz123>` logs.
+.. See if was possible to pull the UnifiedPush image.
+. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+. Check if the secret was created
+.. Go to the namespace of the operator
+.. Run `oc get secrets`
+. Check if the values in the secret are correct.
+.. Run `oc edit secret`
```

Comment:
This would give the values in the secret more succinctly (I presume the postgresql secret is the one being referred to here, please correct me if I'm wrong):

```
for field in POSTGRES_{DATABASE,HOST,USERNAME,PASSWORD,SUPERUSER,VERSION}; do printf "${field}: " && oc get secret <CR Name>-postgresql -o jsonpath="{.data.$field}" | base64 -d && echo; done
```

But even with that, it's not clear to me how a user would even know if those values are correct? Those values are all produced from within the operator, and they're used to both configure the database and to populate the env vars in UPS to be able to connect to the database. Should people just check if they exist? Or should there be more steps to try to connect to the database with these details?